modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -861,15 +862,38 @@ private Map<String, String> _addAction(
 				}
 
 				for (ObjectEntry nestedObjectEntry : nestedObjectEntries) {
```

Comment:
What about custom with System? I think we should apply to both worlds

Reply:
if possible

Reply:
I'm going to take note of this because it is going to be solved on this https://liferay.atlassian.net/browse/LPS-186732

Reply:
As we have talked, it not possible to set this property in the System Object entries because it is not exposed in the schema

Reply:
This happens in the many to one relationship